routes/web.php

```diff
@@ -186,20 +186,20 @@
 
 
 
-//task 3
+//task 3.1
 Route::post('users',[Xcontroller::class,'my_data']);
```

Comment:
Hello Esraa,, 

the functionality of your code is correct, but i have a little note here, you have violated some standard when i download your project in my computer look at this warning message

```shell
Class App\Http\Controllers\xcontroller located in ./app/Http/Controllers/Xcontroller.php does not comply with psr-4 
autoloading standard (rule: App\ => ./app). Skipping.
```

This means you didn't follow PHP standard recommendations

Your class definition
```php
class xcontroller extends Controller
```
correct
```php
class XController extends Controller
```